bin/run

```diff
@@ -17,9 +17,9 @@ enable_stats_monitor = os.environ.get('TFE_MONITOR_STATS') is not None
 enable_traces = os.environ.get('TFE_TRACE') is not None
 enable_debug = os.environ.get('TFE_DEBUG') is not None
 
-tfe.setMonitorStatsFlag(enable_stats_monitor)
-tfe.setTFETraceFlag(enable_traces)
-tfe.setTFEDebugFlag(enable_debug)
+tfe.set_tfe_stats_flag(enable_stats_monitor)
```

Comment:
if we're renaming then let's use a better matching name, say `set_tfe_events_flag` (per https://www.tensorflow.org/guide/summaries_and_tensorboard)